libflux/src/flux/semantic/builtins.rs

```diff
@@ -347,6 +347,8 @@ pub fn builtins() -> Builtins<'static> {
                 "assertEquals" => "forall [t0] (name: string, <-got: [t0], want: [t0]) -> [t0]",
                 "assertEmpty" => "forall [t0] (<-tables: [t0]) -> [t0]",
                 "diff" => "forall [t0] (<-got: [t0], want: [t0], ?verbose: bool) -> [{_diff: string | t0}]",
+                "fail" => "forall [] () -> bool",
+                "yieldValue" => "forall [t0] (<-v: t0) -> t0",
```

Comment:
I don't see where this function is defined?

Reply:
My original idea was to create a special yield function for testing but this didn't work so this is a mistake.